components/card.vue

```diff
@@ -24,24 +48,62 @@
           </span>
         </p>
       </div>
-      <div class="flex flex-row items-center justify-center text-center w-full">
-        <UProgress v-model="normalizedScore" class="my-0 mr-2" size="md" />
-        <p class="text-xs">
-          {{ props.card.score !== undefined
-            ? props.isSimilaritySearch
-              ? `${normalizedScore.toFixed(2)}%`
-              : normalizedScore.toFixed(2) + '%'
-            : 'N/A' }}
-        </p>
+
+      <!-- Action Buttons -->
+      <div class="flex flex-row items-center justify-between text-center w-full">
+        <div class="flex flex-row items-center">
+          <UButton v-if="showCardInfo && card.card_data.tcgplayer_id"
+            :to="getAffiliateLink(card.card_data.tcgplayer_id)" external color="success" variant="solid"
+            class="mt-1 mr-2" icon="i-heroicons-shopping-cart" size="sm" target="_blank" rel="noopener noreferrer">{{
+              card.card_data.prices.usd ? `$${card.card_data.prices.usd}` :
+                'Buy' }}
+          </UButton>
+          <UButton v-if="showCardInfo && !isSearched" color="neutral" variant="solid" class="mt-1 mr-2 cursor-pointer"
+            icon="i-mdi-cards-outline" size="sm" @click="findSimilarCards"></UButton>
+        </div>
+        <div v-if="!isSearched && showCardInfo" class="flex flex-row items-center gap-2">
+          <UTooltip text="I agree with this result!" :popper="{ placement: 'top' }">
```

Comment:
Also, I don't think we need thumbs up as if it's not flagged negatively that implies it's a good enough quality result to make sense. @benchislett seems to agree.